tests/drivers/pwm/pwm_loopback/boards/efr32bg22_brd4184a.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Centro de Inovação EDGE
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	pwm_loopback_0 {
+		compatible = "test-pwm-loopback";
+		pwms = <&pwm0 0 0 PWM_POLARITY_NORMAL>,
+		       <&pwm1 0 0 PWM_POLARITY_NORMAL>;
+	};
+};
+
+&timer1 {
+	status = "okay";
+
+	pwm1: pwm {
+		status = "okay";
+		pin-location = <GECKO_LOCATION(0) GECKO_PORT_B GECKO_PIN(3)>;
```

Comment:
It looks like you're using the legacy way of pinmuxing here. This is unfortunately no longer allow in Zephyr. You should switch to the Pin Control API.